include/defs__texture.h

```diff
@@ -0,0 +1,19 @@
+#ifndef DEFS__TEXTURE_H
+#define DEFS__TEXTURE_H
+
+#define TEX_AMOUNT 4
+#define TEX_WIDTH 64
+#define TEX_HEIGHT 64
+#define WNORTH 0
+#define WSOUTH 1
+#define WEAST 2
+#define WWEST 3
+#define FLOOR 4
+#define CEILING 5
+#define CEILING2 6
```

Comment:
```suggestion
typedef enum e_texdata
{
	TEX__AMOUNT = 4,
	TEX__WIDTH = 64,
	TEX__HEIGHT = 64,
	TEX__NORTH = 0,
	TEX__WALL__SOUTH,
	TEX__WALL__EAST,
	TEX__WALL__WEST,
	TEX__WALL__FLOOR,
	TEX__CEILING0,
	TEX__CEILING1,
}	t_texdata;
```
it would be great if those data could be presented as enums.

Reply:
Done !